homeassistant/components/aquostv/media_player.py

```diff
@@ -209,12 +171,12 @@ def turn_off(self):
     @_retry
     def volume_up(self):
         """Volume up the media player."""
-        self._remote.volume(int(self._volume * 60) + 2)
+        self._remote.volume(int(self.volume_level * 60) + 2)
 
     @_retry
     def volume_down(self):
         """Volume down media player."""
-        self._remote.volume(int(self._volume * 60) - 2)
+        self._remote.volume(int(self.volume_level * 60) - 2)
```

Comment:
volume_level just returns _attr_volume_level, use _attr_volume_level here

Reply:
This is confusing as it is suggested differently: https://developers.home-assistant.io/docs/core/entity/#entity-class-or-instance-attributes (see the tip)

Reply:
I'd argue that is more confusing, it's not viewable in the file you are looking at...
But if it's in the docs, it should be followed 👍